canvas_modules/common-canvas/__tests__/common-properties/controls/checkboxset-test.js

```diff
@@ -31,27 +31,32 @@ const controller = new Controller();
 const control = {
 	name: "test-checkboxset",
 	values: ["apple", "grape", "orange", "pear"],
-	valueLabels: ["apple", "grape", "orange", "pear"]
+	valueLabels: ["apple", "grape", "orange", "pear"],
+	valueDescs: ["desc for 0", "desc for 20"]
 };
 const controlNull = {
 	name: "test-checkboxset-null",
 	values: ["apple", "orange", "pear"],
-	valueLabels: ["apple", "orange", "pear"]
+	valueLabels: ["apple", "orange", "pear"],
+	valueDescs: ["desc for 0", null, "desc for 20", null]
```

Comment:
Why add these?  I don't think the rest of these are validated.

Reply:
Removed unused ones.